misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -153,6 +154,23 @@ class JettyService @Inject internal constructor(
       server.addConnector(httpsConnector)
     }
 
+    if (webConfig.sidecar != null) {
+      val udsConnectionFactories = mutableListOf<ConnectionFactory>()
+
+      udsConnectionFactories += HttpConnectionFactory(httpConfig)
+
+      val udsConnector = UnixSocketConnector(
+        server,
+        null /* executor */,
+        null /* scheduler */,
+        null /* buffer pool */,
+        webConfig.selectors ?: -1,
+        udsConnectionFactories.toTypedArray()
```

Comment:
Here you can shorten to `arrayOf(HttpConnectionFactory(httpConfig))` instead of creating a mutable list and adding to it.

Kotlin has listOf, setOf, mapOf, arrayOf, mutableListOf, mutableSetOf, etc helper functions for creating various types of collections.